src/devices/controller.c

```diff
@@ -18,7 +18,8 @@
 #include "v5_api.h"
 #include "vdml/vdml.h"
 
-#define CONTROLLER_MAX_COLS 15
+#define CONTROLLER_MAX_COLS ( 20U )
```

Comment:
Why was this changed from 15 to 20?

Reply:
Because VEXCode supports columns 0-19:
![image](https://github.com/user-attachments/assets/aa3cfe77-2010-4309-964f-c8bebbfb7ed2)